addons/website_sale/tests/test_website_sale_product.py

```diff
@@ -1,15 +1,53 @@
 # coding: utf-8
+import itertools
+
 from odoo.tests import tagged
 from odoo.addons.website.tools import MockRequest
 from odoo.addons.sale.tests.test_sale_product_attribute_value_config import TestSaleProductAttributeValueCommon
+from odoo.addons.website_sale.controllers.main import WebsiteSale
+
 
 @tagged('post_install', '-at_install')
 class WebsiteSaleProductTests(TestSaleProductAttributeValueCommon):
 
     def setUp(self):
         super().setUp()
+        self.WebsiteSaleController = WebsiteSale()
         self.website = self.env.ref('website.default_website')
```

Comment:
Can we change the setUp to a setUpClass to reduce the number of setups now that there are multiple tests in the file ?